dotcom-rendering/src/web/components/MostViewedFooterLayout.tsx

```diff
@@ -16,30 +18,39 @@ const stackBelow = (breakpoint: Breakpoint) => css`
 	}
 `;
 
-const fixedWidths = css`
+const fixedWidths = (
+	renderAds: boolean | undefined,
```

Comment:
```suggestion
	renderAds?: boolean,
```

is a bit more concise!